### PR TITLE
Add google analytics to prod mode

### PR DESCRIPTION
**Pull request by quantumfusion**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -6,6 +6,18 @@
...
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-9FNRKJ31JM"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+
+      gtag('config', 'G-9FNRKJ31JM');
+    </script>
+    {% endif }
```
Comment (baby-bell) [inline templates/index.html]:
```suggestion
    {% endif %}
```

**Review by baby-bell — CHANGES_REQUESTED** [review]
lgtm with fix below

File: templates/index.html (hunk 1 above)
Comment (quantumfusion) [reply templates/index.html]: Ah. good catch.

**Review by baby-bell — APPROVED** [review]

**Review by charleshong3 — APPROVED** [review]
Google just needed some time to warm up

**Pull request merged**